src/qosmic.cpp

```diff
@@ -39,6 +40,26 @@ int main(int argc, char* argv[])
 	Logger::getInstance()->setLevel(Logger::levelFor(getenv("log")));
 	logInfo(QString("main() : Qosmic (version %1)").arg(QOSMIC_VERSION));
 
+    // Translations for Qt strings 
```

Comment:
The translations are loaded again in the following block.  I don't see the problem solved by this change.
I think this change should be removed from your request.